hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -255,15 +255,20 @@ module top_${top["name"]} #(
 % endfor
 
   // Alert list
-  prim_alert_pkg::alert_tx_t [alert_handler_pkg::NAlerts-1:0]  alert_tx;
-  prim_alert_pkg::alert_rx_t [alert_handler_pkg::NAlerts-1:0]  alert_rx;
+<% alert_handlers = [m["name"] for m in top["module"] if "template_type" in m and m["template_type"] == "alert_handler"] %>\
+% for handler in alert_handlers:
+  prim_alert_pkg::alert_tx_t [${handler}_pkg::NAlerts-1:0]  alert${handler.replace("alert_handler", "")}_tx;
+  prim_alert_pkg::alert_rx_t [${handler}_pkg::NAlerts-1:0]  alert${handler.replace("alert_handler", "")}_rx;
+% endfor
 
 % if not top["alert"]:
-  for (genvar k = 0; k < alert_handler_pkg::NAlerts; k++) begin : gen_alert_tie_off
+%    for handler in alert_handlers:
```

Comment:
Would it make sense to move the alert handler signal declaration into a separate template fragment and just include it here?